layouts/partials/header.html

```diff
@@ -43,55 +43,55 @@ <h3 class="self-intro">{{ .Site.Params.selfintro }}</h3>
     <div class="container">
         <div class="content">
         {{ if .Site.Params.github }}
-            <a href="{{ .Site.Params.github }}" target="_blank" rel="noopener"><img class="icon" src="/img/github.svg" alt="github" /></a>
+            <a href="{{ .Site.Params.github }}" target="_blank" rel="noopener"><i class="fa-brands fa-github" style="color: #000000;"></i></a>
```

Comment:
can you please update the changes in this file to the following form?

```
{{ if .Site.Params.fontawesome }}
  /* use the font-awesome icon */
{{ else }}
  /* use icons from the "img" directory */
{{ end }}
```

we want to keep the original icons. otherwise it would be a breaking change for existing users.